### PR TITLE
Clean up pyanaconda.kickstart

### PR DESCRIPTION
**Pull request by poncovka**
Move storage-related functions from `pyanaconda.kickstart`.

### CODE REVIEW

File: pyanaconda/storage/execution.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
     MOUNT_POINT_REFORMAT, MOUNT_POINT_FORMAT, MOUNT_POINT_PATH, MOUNT_POINT_FORMAT_OPTIONS, \
     MOUNT_POINT_MOUNT_OPTIONS
 from pyanaconda.core.i18n import _
-from pyanaconda.kickstart import refreshAutoSwapSize, getEscrowCertificate, getAvailableDiskSpace, \
+from pyanaconda.kickstart import refreshAutoSwapSize, getAvailableDiskSpace, \
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/storage/execution.py]: Not in this commit. This line is removed completely in this pull request.

File: pyanaconda/storage/autopart.py [hunk 2]
```diff
@@ -44,6 +44,19 @@
...
+    :param storage: blivet.Blivet instance
+    :return: overall disk space available
+    :rtype: :class:`blivet.size.Size`
+    """
+    free_space = storage.free_space_snapshot
+    # Blivet creates a new free space dict to instead of modifying the old one,
+    # so there is no worry about the dictionary changing during iteration.
+    return sum(disk_free for disk_free, fs_free in free_space.values())
+
+
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/storage/autopart.py]: Fixed.

File: pyanaconda/storage/autopart.py [hunk 3]
```diff
@@ -493,6 +493,14 @@ def do_autopart(storage, data, min_luks_entropy=None):
...
             :meth:`pyanaconda.storage.InstallerStorage.clear_partitions`.
     """
+    # Refresh size of the auto partitioning request for swap device according
+    # to the current state of the storage configuration.
+    for request in storage.autopart_requests:
+        if request.fstype == "swap":
+            disk_space = get_available_disk_space(storage)
+            request.size = swap_suggestion(disk_space=disk_space)
+            break
+
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/storage/autopart.py]: Fixed.

**Comment by poncovka** [comment]
Updated. The errors are not related to this pull request.

File: pyanaconda/storage/execution.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkonecny12) [reply pyanaconda/storage/execution.py]: OK

**Comment by poncovka** [comment]
Rebased.

**Pull request merged**